[CES-324] Added example and documentation for a develop environment with Core infra (#261)
pagopa/dx: infra/modules/azure_core_infra/README.md

}
```

> For more specific examples check out the `./example` folder, here is possible to find a **complete** example that creates a core infrastructure and a **develop** example that in addition to creating the basic infrastructure will create additional resources useful for a development environment.

Reviewer: ```suggestion
For more specific examples check out the `./example` folder. There you'll find find a "complete" example that creates the core infrastructure and a "develop" example that, in addition to creating the basic infrastructure, will configure additional resources useful to bootstrap a development environment on Azure.
```
